src/pages/en/guides/migration-guide/from-gatsby.md

```diff
@@ -0,0 +1,1184 @@
+
+- Inline styles
+- Component-specific styling
+- Global styling
+
+#### Inline Styles
+
+Convert any inline style objects in React (`style={{fontWeight: "bold", }}`) to inline HTML style attributes (`style="font-weight:bold;"`).
+
+```jsx
+// JSX
+<p style={{fontWeight: "bold"}}>Hello, world</p>
+```
+
+```astro
+// Astro
+<p style="font-weight: bold;">Hello, world</p>
+```
+
+#### Component-specific styling
+
+One of Astro's unique capabilities enables you to use standard `<style>` tags for scoped CSS styles.
+
+```astro
+<p>Hello, world</p>
+
+<!-- The following will only apply to this component -->
+<style>
+p {
+   font-weight: bold;
+}
+</style>
+```
+
+However, you can extract your styling to a [CSS Module file](https://docs.astro.build/en/guides/imports/#css-modules) and import that instead.
+
+```astro title="src/components/Hello.astro"
+---
+import styles from './Hello.module.css';
+---
+
+<p>Hello, world</p>
+```
+
+ ```css title="src/components/Hello.module.css"
+ p {
+    font-weight: bold;
+ }
+ ```
+
+#### Global Styling
+
+Global styling is applied similarly between Gatsby and Astro by importing a `css` file within a layout file.
+
+```astro title="src/layouts/Layout.astro" {2}
+---
+import "global.css";
+---
+
+<html>
+  <head>
+    <meta charset="utf-8" />
+	<link rel="icon" type="image/svg+xml" href="/favicon.svg" />
+	<meta name="viewport" content="width=device-width" />
+	<meta name="generator" content={Astro.generator} />
+	<title>Astro</title>
+  </head>
+  <body>
+    <slot/>
+  </body>
+</html>
+```
+
+```css title="src/layouts/global.css"
+p {
+    font-weight: bold;
+}
+```
+
+#### CSS Preprocessors
+
+Astro supports installing and using many [CSS preprocessors](/en/guides/styling/#css-preprocessors).
+
+As an example, to use Sass, you would run:
+
+```shell
+npm install -D sass
+```
+
+After doing so, you're then able to import `.scss` or `.sass` files as you would a `.css` file. Additionally, you're able to declare the CSS preprocessor used to modify `style` tags:
+
+```astro title="src/layouts/Layout.astro"
+<p>Hello, world</p>
+
+<style lang="scss">
+p {
+   color: black;
+   
+   &:hover {
+       color: red;
+   }
+}
+</style>
+```
+
+### Gatsby Code Comments to Astro
+
+An Astro file uses JavaScript code comments in the frontmatter `//` but HTML code comments in the template body. (`<!-- -->`)
+
+### Gatsby Image Plugin to Astro
+
+Astro provides [a native Image integration](https://docs.astro.build/en/guides/images/#astros-image-integration) for optimizing and working with images. This can be installed into an existing Astro project using the command line, and provides an `<Image />` component to finely control the display of a single image and a `<Picture />` component for responsive images in any `.astro` or `.mdx` file. You will need to replace Gatsby's `<StaticImage />` and `<GatsbyImage />` components with these components (and update required attributes), or with an HTML `<img>` tag. 
```

Comment:
```suggestion
Astro provides [a native Image integration](https://docs.astro.build/en/guides/images/#astros-image-integration) for optimizing and working with images. This can be installed into an existing Astro project using the command line, and provides an `<Image />` component to finely control the display of a single image and a `<Picture />` component for responsive images in any `.astro` or `.mdx` file. 

You will need to replace Gatsby's `<StaticImage />` and `<GatsbyImage />` components with these components (and include any required attributes), or with an HTML `<img>` tag. 
```